### ISSUE TITLE
Response from Rating Returns single RatedPackage object instead of an array of RatedPackage as per spec

### ISSUE THREAD

**Issue by ericvandamme** [UPS-API/api-documentation#14]
As per the [API Reference](https://developer.ups.com/api/reference?loc=en_US#operation/Rate!c=200&path=RateResponse/RatedShipment/RatedPackage&t=response), I should get an Array of objects (Rate_RatedShipment_RatedPackage), yet I get just a single object instead failing on deserialization.

Example:
```
{
	"RateResponse":{
		"Response":{
			"ResponseStatus":{
				"Code":"1", 
				"Description":"Success"
			}, 
			...
		}, 
		"RatedShipment":[{
			"RatedPackage":{
				"Weight":"1.0"
			},
			...
```

**Comment by ericvandamme** [comment]
Additionally seems to affect `object (Rate_TimeInTransit_ServiceSummary)`, except that it is right in the documentation, but incorrect in the OpenAPI json spec.

**Comment by elidrissidev** [comment]
This issue seems to be affecting several endpoints, see https://github.com/UPS-API/api-documentation/issues/3.

**Comment by SpitfireMike** [comment]
ShipmentResponse/ShipmentResults/PackageResults also.  Documented as Array of objects (Ship_ShipmentResults_PackageResults), but only getting a single .

**Comment by ericvandamme** [comment]
> ShipmentResponse/ShipmentResults/PackageResults also. Documented as Array of objects (Ship_ShipmentResults_PackageResults), but only getting a single .

Just so you know, I published my solution for everyone in issue #3

**Comment by UPSRahulBhatnagar** [comment]
Hi, thank you for your comment. Please refer to: https://github.com/UPS-API/api-documentation/issues/3#issuecomment-1796167729. We will keep the linked thread open for discussion.
